safeeyes/core.py

```diff
@@ -184,40 +168,51 @@ def __take_break(self, break_type = None):
         utility.execute_main_thread(self.__fire_start_break)
 
     def __scheduler_job(self):
-        """
-        Scheduler task to execute during every interval
-        """
+        """Scheduler task to execute during every interval."""
         if not self.running:
             return
 
         current_time = datetime.datetime.now()
         current_timestamp = current_time.timestamp()
 
-        if self.context['state'] == State.RESTING and self.paused_time > -1:
+        if self.context["state"] == State.RESTING and self.paused_time > -1:
```

Comment:
Magic numbers like `-1` is better to move to constants

Reply:
Good suggestion. Is there a linter rule that could help with that?

Reply:
Probably [this](https://docs.astral.sh/ruff/rules/magic-value-comparison) rule